Add new rule: `no-class-bindings` (#1769)
ember-template-lint/ember-template-lint: docs/rule/no-class-bindings.md

# no-class-bindings

It is possible to pass `classBinding` and `classNameBindings` as arguments to a component when invoked with curly syntax.

```hbs
{{some-component classNameBindings="foo:truthy:falsy"}}
```

> These arguments are merged into the class attribute on the class, regardless of whether or not the component is a classic component which contains the classNameBindings logic.

Author: I'm open to a better description. This one sounds janky.

> These arguments are merged into the class attribute on the class, regardless of whether or not the component is a classic component which contains the classNameBindings logic.

Reviewer: ```suggestion
These arguments are merged into the `class` attribute, whether or not the component is a classic component (which contains the `classNameBindings` logic).
```

> This rule disallows components accepting either `classBinding` or `classNameBindings`.

Reviewer: ```suggestion
This rule disallows use of `classBinding` or `classNameBindings` on components.
```
